catalog/ui/catalog-ui-search/src/main/webapp/component/query-interactions/query-interactions.hbs

```diff
@@ -18,6 +18,14 @@
         Run
     </div>
 </div>
+<div class="query-interaction interaction-hit-count" title="Hit Count" data-help="Executes the search returning only the hit count">
+    <div class="interaction-icon">
+        <b>#</b>
```

Comment:
❗️ Recommend updating this icon to be a "Refresh" icon like https://fontawesome.com/icons/sync-alt?style=solid to be more aligned to the future version of Intrigue (see Teresa for mockups).